src/os/webgl/webgl.js

```diff
@@ -48,3 +48,29 @@ os.webgl.hasPerformanceCaveat = function() {
   }
   return null;
 };
+
+/**
+ * An altitude mode id and name
+ * @typedef {Object} AltitudeModeValue
```

Comment:
Should I redo this line to be `@typedef {Object} os.webgl.AltitudeModeValue` or is it already scoped here? It's use below as the enum type is from line 58 below

Reply:
Currently OpenSphere uses [this format](https://github.com/ngageoint/opensphere/blob/master/src/os/color.js#L10-L17) for `@typedef`, so please use that for consistency.